alf/utils/dist_utils.py

```diff
@@ -654,14 +654,14 @@ def _get_transform_builder(transform):
         return transform.__class__
 
     def _get_transform_params(transform):
-        if hasattr(transforms, 'params') and transforms.params is not None:
+        if hasattr(transform, 'params') and transform.params is not None:
```

Comment:
This bug seems to be a serious one that affect the training. @hnyu, does your previous experiments on normalized flow depend on this?

Reply:
It's my understanding that this is more of a silent bug as the logic of the function does not technically change.

Reply:
> This bug seems to be a serious one that affect the training. @hnyu, does your previous experiments on normalized flow depend on this?

I also believe it's just a minor bug. If you look at the code, when _get_transform_params is called, `transform` is always the same with `transforms`